README.md

```diff
@@ -71,19 +71,34 @@ devtools::install_github("tensorflow/io", subdir = "R-package")
 
 ### Python
 
-The TensorFlow I/O package (`tensorflow-io`) could be built from source:
+For Python development, a reference Dockerfile [here](dev/Dockerfile) can be
+used to build the TensorFlow I/O package (`tensorflow-io`) from source:
 ```sh
-$ docker run -it -v ${PWD}:/working_dir -w /working_dir tensorflow/tensorflow:custom-op
-$ # In docker
-$ curl -OL https://github.com/bazelbuild/bazel/releases/download/0.20.0/bazel-0.20.0-installer-linux-x86_64.sh
-$ chmod +x bazel-0.20.0-installer-linux-x86_64.sh
-$ ./bazel-0.20.0-installer-linux-x86_64.sh
+$ # Build and run the Docker image
+$ docker build -f dev/Dockerfile -t tfio-dev .
+$ docker run -it -rm --net=host -v ${PWD}:/v -w /v tfio-dev
+$ # In Docker, configure will install TensorFlow or use existing install
 $ ./configure.sh
-$ bazel build build_pip_pkg
-$ bazel-bin/build_pip_pkg artifacts
+$ # Build TensorFlow I/O C++
+$ bazel build -s --verbose_failures //tensorflow_io/...
+$ # Run tests with PyTest, note: some tests require launching additional containers to run (see below)
+$ pytest tests/
+$ # Build the TensorFlow I/O package
+$ python setup.py bdist_wheel
```

Comment:
I think `python setup.py --data bazel-bin bdist_wheel` might be needed. Otherwise the PR looks good to me.

Reply:
Thanks @yongtang for the reveiw!  Actually, the `--data bazel-bin` option is not needed when working in the container since the env var `TFIO_DATAPATH` is set. This should be added as a note though.